dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -60,7 +58,7 @@
   private static final HelpFormatter helpFormatter = new HelpFormatter();
 
   private final CommandLine commandLine;
-  private ImmutableList<Path> cachedInputClasspath;
+  private ClassPathResult classPathResult;
```

Comment:
It feels very strange that there's a classpath result hiding inside the command line arguments class. Is there a more appropriate abstraction here?

Reply:
I've been feeling that strangeness. Let me update them.